kafka/connection/README.md

```diff
@@ -0,0 +1,18 @@
+# Kafka Activity
+
+This connection connects to kafka cluster.
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/messaging-contrib/kafka/connection
+```
+
+## Configuration
+
+### Settings: 
+| Name       | Type   | Description
+|:---        | :---   | :---   
+| brokerUrls | string | The brokers of the Kafka cluster to connect to - ***REQUIRED***
+| user       | string | If connecting to a SASL enabled port, the user id to use for authentication
```

Comment:
should user, pass and truststore be part of connection? Maybe have a connection section with settings?